server/auth/jwt_test.go

```diff
@@ -132,6 +134,68 @@ func testJWTInfo(t *testing.T, opts map[string]string) {
 	}
 }
 
+func TestJWTNoUsernameRevision(t *testing.T) {
+	optsMap := map[string]map[string]string{
+		"RSA": {
+			"pub-key":     jwtRSAPubKey,
+			"priv-key":    jwtRSAPrivKey,
+			"sign-method": "RS256",
+		},
+	}
+
+	t.Run("RSA", func(tt *testing.T) {
```

Comment:
Please use table driven golang sub test to test the following two cases:
1. Only  has username and exp;
2. Only has revision and exp;

Reply:
Please refer to example https://github.com/etcd-io/etcd/blob/caa775d23563efdec9b4fa6519fdfbfe30985e5f/server/etcdserver/server_test.go#L67